config/initializers/cookies_serializer.rb

```diff
@@ -2,4 +2,4 @@
 
 # Specify a serializer for the signed and encrypted cookie jars.
 # Valid options are :json, :marshal, and :hybrid.
-Rails.application.config.action_dispatch.cookies_serializer = :marshal
+Rails.application.config.action_dispatch.cookies_serializer = :json
```

Comment:
```suggestion
Rails.application.config.action_dispatch.cookies_serializer = :hybrid
```

From this conversation it sounds better to first switch to `:hybrid` and then switch to `:json` a bit later: https://gds.slack.com/archives/CAB4Q3QBW/p1564042299039500

https://blog.bigbinary.com/2014/12/23/migrating-existing-session-cookies-while-upgrading-to-rails-4-1-and-above.html